osidb/models.py

```diff
@@ -832,6 +832,30 @@ def _validate_notabug_flaw_affected(self):
                 f"{affected.ps_component} is affected by a flaw solved as NOTABUG.",
             )
 
+    def _validate_incomplete_errata_comment(self):
+        """
+        Alerts in case comments are missing affected product(s) for a released erratum
```

Comment:
I think we need to include in these docstrings the reasoning behind the business logic, I don't quite understand what's going on here so I don't know whether the body is right or wrong

Reply:
Added full description from SFM2 for clarification.

Reply:
That helps, thanks